procrastinate/retry.py

```diff
@@ -97,6 +154,16 @@ def get_schedule_in(self, *, exception: BaseException, attempts: int) -> int | N
         wait += self.exponential_wait ** (attempts + 1)
         return wait
 
+    def get_retry_decision(
+        self, *, exception: BaseException, job: Job
+    ) -> RetryDecision:
+        schedule_in = self.get_schedule_in(exception=exception, attempts=job.attempts)
+
+        if schedule_in is None:
+            return RetryDecision(should_retry=False)
+
+        return RetryDecision(should_retry=True, schedule_in=schedule_in)
```

Comment:
```suggestion
        return RetryDecision(should_retry=schedule_in is not None, schedule_in=schedule_in)
```
(If too ugly, store `should_retry` in an intermediate variable)

Reply:
As discussed above, I will remove `should_retry` and allow `get_retry_decision` to return `None` (which means no retry then). Ok?

Reply:
Yep !